sandy/core/endf6.py

```diff
@@ -266,8 +271,17 @@ def get_endf6_file(library, kind, zam, to_file=False):
         text = "".join([foo_read(name, url) for name in files.values()])
         tape = Endf6.from_text(text)
     else:
-        filename = files[zam]
-        tape = foo_get(filename, url)
+        if type(zam) == pd.Series:
```

Comment:
I expected something like this example:

inputs
-------
```
url = sandy.libraries.URL_DECAY_JEFF_33_IAEA
files = sandy.libraries.DECAY_FILES_JEFF_33_IAEA
zam = (551340, 551330, 601480)
foo_get = sandy.Endf6.from_zipurl
```
what I would do
---------------
```
if hasattr(zam, "__len__"):
    tapes = map(lambda x: foo_get(files[x], url), zam)
    tape = reduce(lambda x, y: x.add_sections(y.data), tapes)
else:
    tape = foo_get(files[zam], url)
```
this should work both for scalar or iterables...lists, tuples, arrays, series, map objects, filter objects and so on.

It is taken from `Endf6.reduce`, as i mentioned to you!
Please check it before implementing it!